src/name/mlopatkin/andlogview/MainFrame.java

```diff
@@ -261,14 +265,14 @@ public void windowClosing(WindowEvent e) {
     }
 
     private void setupSearchButtons() {
-        bindKeyGlobal(KEY_SHOW_SEARCH_FIELD, ACTION_SHOW_SEARCH_FIELD, new AbstractAction() {
+        bindKeyGlobal(KeyEvent.VK_F, SHORTCUT_KEY_MASK, ACTION_SHOW_SEARCH_FIELD, new AbstractAction() {
```

Comment:
I propose to add an overload that takes `KeyStroke` and just create `private static final KeyStroke KEY_SHOW_SEARCH_FIELD` instead, it would be easier at this point